docs/index.rst

```diff
@@ -17,9 +17,7 @@ If you cannot upgrade South then you need to set SOUTH_MIGRATION_MODULES_.
 
 .. code-block:: python
 
-   SOUTH_MIGRATION_MODULES = {
-      'djedi': 'djedi.south_migrations'
-   }
+   SOUTH_MIGRATION_MODULES = {"djedi": "djedi.south_migrations"}
```

Comment:
```suggestion
   SOUTH_MIGRATION_MODULES = {
       "djedi": "djedi.south_migrations",
   }
```

But, this probably needs to change in the docs to not mention `South`?

Reply:
> this probably needs to change in the docs to not mention South?

What do you want?